### PR TITLE
Fixing styled components

### PR DESCRIPTION
**Pull request by laithAwad**
I have removed an import for "styled components" that wasn't being used and caused my system to stop compiling the frontend part. after discussing and debugging with team members we found the problem in /Client/src/Components/DataCell/DataCell.js and removed the import line.

### CODE REVIEW

**Review by GD-Dheer — APPROVED** [review]
Thanks

File: Client/src/Components/DataCell/DataCell.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
-import React from 'react';
-import Box from '@material-ui/core/Box';
+import Box from '@material-ui/core/Box'
 import Button from '@material-ui/core/Button'
-import styled from 'styled-components';
+import { Container } from '@material-ui/core'
+import React from 'react'
+import styled from 'styled-components'
```
Comment (TiraFire) [inline Client/src/Components/DataCell/DataCell.js:5]: This should be gone! Why is it back XD

**Review by TiraFire — CHANGES_REQUESTED** [review]
There is still the same problem!

**Comment by TiraFire** [comment]
This is a wrong pull request, it merges the develop branch into master. I assume this was an accident with new technology (gitkraken). We will have you create a new pull request from a new bug branch into develop, then close this PR without merging. @laithAwad  @GD-Dheer

**Pull request closed without merging**